frontend/src/signup/components/AddressFields.jsx

```diff
@@ -4,11 +4,18 @@ import { InputField } from './forms';
 
 const AddressFields = () => (
   <fieldset>
-    <Field component={InputField} id="address" name="postalAddress[address]" label="Street address" type="text" />
-    <Field component={InputField} id="suburb" name="postalAddress[suburb]" label="Suburb" type="text" />
-    <Field component={InputField} id="state" name="postalAddress[state]" label="State" type="text" />
-    <Field component={InputField} id="postcode" name="postalAddress[postcode]" label="Postcode" type="text" />
-    <Field component={InputField} id="country" name="postalAddress[country]" label="Country" type="text" />
+    <Field
+      optional
+      component={InputField}
+      id="address"
+      name="postalAddress[address]"
+      label="Street address"
+      type="text"
+    />
+    <Field optional component={InputField} id="suburb" name="postalAddress[suburb]" label="Suburb" type="text" />
+    <Field optional component={InputField} id="state" name="postalAddress[state]" label="State" type="text" />
+    <Field optional component={InputField} id="postcode" name="postalAddress[postcode]" label="Postcode" type="text" />
+    <Field optional component={InputField} id="country" name="postalAddress[country]" label="Country" type="text" />
```

Comment:
Just made all these optional so they don't get the asterisk. Which made the first one go > 120.